app.js

```diff
@@ -177,6 +178,14 @@ History.prototype.toJSON = function () {
     return this.facts.reverse().toJSON();
 };
 
+function buildFinalTRpath(baseURL, thisVersionURL) {
+
+    var w3URL = /https?:\/\/(www\.)?w3\.org\/tr/i;
```

Comment:
@tripu, the registration should not work with `https://www.w3.org/TR/...`, `http://WWW.w3.org/tr/...`, http://w3.org/TR/...`, etc.
The only accepted this version is `http://www.w3.org/TR/...`.